src/bot/root_pythia_cogs.py

```diff
@@ -14,8 +14,8 @@
 
 class RootPythiaCommands(commands.Cog, name=NAME):
     """
-    Define the commands that the bot will respond to, prefixed with the `command_prefix` defined at
-    the bot init
+    Define the commands that the bot will respond to, prefixed with '!' \
```

Comment:
I agree with this. Yet when the help message is shown, it would be better to actually see the prefix and not this `command_prefix` which is not helpful at all.

I wanted to do a f-string but couldn't on a `"""` string :/ Do you have any solutions for that ?

Reply:
https://www.w3schools.com/python/ref_string_format.asp

Reply:
> I wanted to do a f-string but couldn't on a """ string :/ Do you have any solutions for that

Seems to me that you can't here, because this """ string is not proper code, it won't be executed 

Maybe by modifying the help command? at a point it must retrieves this string and then you'll be able to fromat it. 

But in order not to slow this issue again, you coild also create a clear and precise Issue about this, at least for tracking purposes.
If you choose this solution you can resolve this :wink:

Reply:
@J3y0 maybe we'd need your pov on this before merging!

Reply:
@J3y0 I netoiced https://github.com/iScsc/RootPythia/issues/31#issuecomment-1779998251 the pb in the appropriate issue we'll solve it in due time, for now lets resolve this and merge